### ISSUE TITLE
使用ray训练PPO时，需要把模型都上传到集群吗，为什么我每次运行的时候ray都会因为要传输的东西太大而断开连接

### ISSUE THREAD

**Issue by hehebamei** [OpenRLHF/OpenRLHF#399]
![图片](https://github.com/user-attachments/assets/0c04eb59-32a6-492d-bf35-549a3ed3ea9c)
启动的sh：
set -x 
export PATH=$HOME/.local/bin/:$PATH

ray job submit --address="http://127.0.0.1:8265" \
   --runtime-env-json='{"working_dir": "/openrlhf", "pip": "/openrlhf/requirements.txt"}' \
   -- python3 examples/train_ppo_ray.py \
   --ref_num_nodes 1 \
   --ref_num_gpus_per_node 1 \
   --reward_num_nodes 1 \
   --reward_num_gpus_per_node 1 \
   --critic_num_nodes 1 \
   --critic_num_gpus_per_node 1 \
   --actor_num_nodes 1 \
   --actor_num_gpus_per_node 1 \
   --vllm_num_engines 2 \
   --vllm_tensor_parallel_size 2 \
   --colocate_critic_reward \
   --colocate_actor_ref \
   --ref_reward_offload \
   --pretrain OpenRLHF/Llama-3-8b-sft-mixture  \
   --reward_pretrain OpenRLHF/Llama-3-8b-rm \
   --save_path ./checkpoint/llama-3-8b-rlhf \
   --micro_train_batch_size 1 \
   --train_batch_size 128 \
   --micro_rollout_batch_size 1 \
   --rollout_batch_size 1024 \
   --max_samples 100000 \
   --max_epochs 1 \
   --prompt_max_len 1024 \
   --generate_max_len 1024 \
   --zero_stage 3 \
   --bf16 \
   --actor_learning_rate 5e-7 \
   --critic_learning_rate 9e-6 \
   --init_kl_coef 0.01 \
   --prompt_data OpenRLHF/prompt-collection-v0.1 \
   --input_key context_messages \
   --apply_chat_template \
   --normalize_reward \
   --adam_offload \
   --flash_attn \
   --gradient_checkpointing \
   --use_wandb {7fed3f8dc9169294c9f63d05a29295b1e271b0b6}

**Comment by hehebamei** [comment]
![1722569826080](https://github.com/user-attachments/assets/4972daf6-df70-4ddc-b36a-e23d619609de)
文件的结构

**Comment by hijkzzz** [comment]
不要把模型放到项目目录下
